kamel-core/src/jsMain/kotlin/io/kamel/core/cache/httpCache.js.kt

```diff
@@ -0,0 +1,5 @@
+package io.kamel.core.cache
+
+import io.ktor.client.plugins.cache.storage.CacheStorage
+
+internal actual fun httpCacheStorage(maxSize: Long): CacheStorage = CacheStorage.Disabled
```

Comment:
Yeah js web doesn't have okio filesystem support. So that's fine but any thoughts on how it could be implemented? I know https://github.com/russhwolf/multiplatform-settings wraps https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API

Reply:
I am not much familiar with kotlin/js or kotlin/asm. We need some `okio` FS layer on top of the browser storage API to make this work. Also, modern browsers automatically cache the get requests so it might not even be necessary.